README.md

```diff
@@ -18,15 +18,41 @@ Or with bundler,
 gem "hubspot-ruby"
 ```
 
-## Authentication with an API key
+## Getting Started
+This library can be configured to use OAuth or an API key. To find the
+appropriate values for either approach, please visit the [HubSpot API
+Authentication docs].
+
+Below is a complete list of configuration options with the default values:
+```ruby
+Hubspot.configure({
+  hapikey: <HAPIKEY>,
```

Comment:
I've just opened a PR on the node side adding these to the readme with values that are closer to reality. Do you think it's worth doing something similar here? https://github.com/MadKudu/node-hubspot/pull/138

Reply:
Your README updates in the node project look excellent 🌟 I think this README would benefit from additional information around the "contributing" and "testing" sections. The list of configuration options here is more for the API users rather than a contributor. 

This idea is based on issue #18 where a user had trouble understanding how to set the `portal_id` value.

Reply:
I do like how you've included examples of the access token and api key. I worry if I add those values here, it will conflict with showing the default values for `base_url` and `logger`. I'm going to defer adding that for now and add a similar example when updating the contributing section,